source/_drafts/radio-notification.md

```diff
@@ -0,0 +1,270 @@
+
+// Primary Layout View.
+const Layout = View.extend({
+  template: template('<div id="notify"></div>'),  // Add hooks for rest of app.
+
+  regions: {
+    notification: NotificationRegion  // Include other regions here.
+  }
+});
+
+
+const ExampleApp = Application.extend({
+  region: '#application',
+
+  onStart() {
+    this.showView(new Layout());  // Initialize everything.
+  }
+});
+
+
+// Start the application.
+const application = new ExampleApp();
+application.start();
+```
+
+I've skipped over the rest of the application. You'll need to include regions
+for the rest of your application e.g. navbars, the main view for this to be
+fully functional.
+
+
+## Using Notifications
+
+Now, to use this system, you can just access the Radio channel from anywhere in
+your app and the `NotificationRegion` will show notifications based on your
+input. Let's see a simple example to show a model save failed:
+
+```js
+import {template} from 'underscore';
+
+import {View} from 'backbone.marionette';
+import Radio from 'backbone.radio';
+
+
+export const ExampleView = View.extend({
+  tagName: 'form',
+
+  template: template('<input name="username" id="id_username"><button>Save</button>'),
+
+  ui: {
+    username: '#id_username'
+  },
+
+  events: {
+    submit: 'saveForm'
+  },
+
+  modelEvents: {
+    sync: 'saveSuccessful',
+    error: 'error'
+  },
+
+  saveForm() {
+    this.model.save({
+      username: this.getUI('username').val()
+    });
+  },
+
+  saveSuccessful() {
+    const channel = Radio.channel('notify');
+    channel.request('show:success', 'User created successfully');
+  },
+
+  error() {
+    const channel = Radio.channel('notify');
+    channel.request('show:error', 'An error occurred creating the user');
+  }
+});
+```
+
+Now, whenever you save your form, the application will either show a success or
+error message based on the server response. You could add an `invalid` hook as
+well if your model includes client-side validation.
+
+## Conclusion
+
+You should now have a good grasp on how you can use the Radio as a global
+message bus to show notifications in a totally unrelated part of your app. This
+pattern isn't constrained to just notifications - anywhere you need to send/receive
+information across your application is ripe for the Radio pattern.
+
+### Future Improvements
+
+Some potential improvements to try out yourself:
+
+* Use a [`Behavior`][behavior] to make it easy for multiple views to respond to form saving
+* Use a `Collection` and [`CollectionView`](collectionview) to show multiple notifications at once
+
+## Get in Touch!
+
+As always, if you have any questions, pop in on [Gitter][gitter] and we'll be
+happy to help!
+
+[behavior]: http://marionettejs.com/docs/v3.1.0/marionette.behavior.html
+[collectionview]: http://marionettejs.com/docs/v3.1.0/marionette.collectionview.html
+[radio]: http://marionettejs.com/docs/v3.1.0/backbone.radio.html
+[region]: http://marionettejs.com/docs/v3.1.0/marionette.region.html
```

Comment:
I know that it's used in your reps app, don't you wanna to add reference on it? 😃